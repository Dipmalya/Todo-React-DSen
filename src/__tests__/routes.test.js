import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../routes';

describe('Test suite to check route is rendered', () => {
    it('Test case to check application snapshot matches', () => {
        const Route = shallow(<Routes />);
        expect(Route).toMatchSnapshot();
    })
})