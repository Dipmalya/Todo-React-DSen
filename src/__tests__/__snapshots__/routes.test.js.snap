// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite to check route is rendered Test case to check application snapshot matches 1`] = `ShallowWrapper {}`;
